lib/Models/GeoJsonCatalogItem.js

```diff
@@ -511,10 +513,21 @@ function loadGeoJson(geoJsonItem) {
     for (var i = 0; i < entities.length; ++i) {
       var entity = entities[i];
 
-      /* If no marker symbol was provided but Cesium has generated one for a point, then turn it into
-               a filled circle instead of the default marker. */
       var properties = entity.properties || {};
-      if (
+
+      // If we've got a marker url use that in a billboard
+      if (defined(entity.billboard) && defined(options.markerUrl)) {
+        entity.billboard = {
+          image: proxyCatalogItemUrl(geoJsonItem, options.markerUrl),
+          heightReference: HeightReference.RELATIVE_TO_GROUND,
+          width: style["marker-width"],
+          height: style["marker-height"],
+          rotation: style["marker-angle"]
+        };
```

Comment:
Why do we need HeightReference here but not elsewhere?

Reply:
Because for these markers we're moving to a billboard and we want them to not float on some unknown part of the atmosphere.

Reply:
re `inheritStyleInformation` is perhaps a better name `useStyleInformationFromService` @steve9164 ?

Reply:
Looks like we should add the height reference to all entities